src/data/reducers/courseChecklist.test.js

```diff
@@ -7,11 +7,11 @@ let state;
 
 describe('Course Checklist Reducers', () => {
   describe('courseBestPractices reducer', () => {
-    it('returns correct state for REQUEST_COURSE_QUALITY_SUCCESS action', () => {
+    it('returns correct state for REQUEST_COURSE_BEST_PRACTICES_SUCCESS action', () => {
```

Comment:
Just renaming the functions to be in line with our decision on naming.

Reply:
Thanks for catching all of these